numpy/random/mtrand/mtrand.pyx

```diff
@@ -573,283 +573,6 @@ def _shape_from_size(size, d):
            shape = tuple(size) + (d,)
     return shape
 
-
```

Comment:
These function should have been already been removed with the creation of the `randint_helpers.pxi.in` file. Sorry that I didn't catch that, please make a PR to do that first.